packages/datadog-plugin-gcloud-pubsub/test/index.spec.js

```diff
@@ -0,0 +1,153 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const { expectSomeSpan, withDefaults } = require('../../dd-trace/test/plugins/helpers')
+const plugin = require('../src')
+
+const BASE_PROJECT_ID = `test-project-${bigRandom()}-`
```

Comment:
Why the trailing hyphen? Also, why not use the existing `id` instead that is able to generate both hex and int?

Reply:
> Why the trailing hyphen?

We're going to add an additional counter number at the end. See the functions at the end of the test file.

> Also, why not use the existing id instead that is able to generate both hex and int?

I had not seen that. Where is it?

Reply:
Replaced with id.

Reply:
Still not sure I understand why the sequential ints at the end.

Reply:
Explained below.